### ISSUE TITLE
Meteor.userId() has value but user is not logged in (on mobile chrome)

### ISSUE THREAD

**Issue by sirpy** [meteor/meteor#9191]
this happens to me on meteor 1.5.2.1 using react, when i browse to the website on a mobile with chrome
I print inside a compenentdidmount Meteor.userId(),Meteor.user() and Meteor.loggingIn() and Meteor.loggingOut()
the result is that i get a value for meteor.userId() even though user is not logged in
```
MainLayout componentDidMount B5F4HMYqabJcdStvE undefined false false
```
then In console i do Meteor.userId() and i get null.

this happens only when opening a new tab, if i refresh the page it works as expected and userId is null on componentdidmount.
on desktop it also works as expected.

I did some further testing and cleared localstorage. and then it also works as expected. so i guess something in localstorage is causing this. resume tokens or something like that.

**Comment by rj-david** [comment]
Please note that there was an account related bug in 1.5.2.1 that was fixed for 1.5.2.2 :
Fixes a regression in accounts-base caused by changes to the (now deprecated) connection.onReconnect function which caused users to be logged out shortly after logging in

**Comment by hwillson** [comment]
As mentioned in https://github.com/meteor/meteor/issues/9191#issuecomment-335000588 (thanks @rj-david!) please try using Meteor 1.5.2.2 (and report back if this issue is still happening). Thanks!
